framework/auth/oauth_scopes.py

```diff
@@ -17,29 +17,29 @@ class CoreScopes(object):
     """
     The smallest units of permission that can be granted- all other scopes are built out of these.
     Each named constant is a single string."""
-    USERS_READ = 'users+read'
-    USERS_WRITE = 'users+write'
+    USERS_READ = 'users_read'
```

Comment:
Doesn't really matter in a practical sense, so we're going to pass the buck for now. See note here: 
https://github.com/CenterForOpenScience/osf.io/pull/4327#issuecomment-142124025